ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerDefinitionExpression/FeatureLayerDefinitionExpression.cpp

```diff
@@ -84,8 +88,40 @@ bool FeatureLayerDefinitionExpression::layerInitialized() const
   return m_initialized;
 }
 
-void FeatureLayerDefinitionExpression::setDefExpression(QString whereClause)
+int FeatureLayerDefinitionExpression::currentFeatureCount_() const
 {
-  // In QML, "req_Type = \'Tree Maintenance or Damage\'"
+  return m_currentFeatureCount;
+}
+
+void FeatureLayerDefinitionExpression::setDefExpression(const QString& whereClause)
+{
+  // In QML, "req_type = \'Tree Maintenance or Damage\'"
   m_featureLayer->setDefinitionExpression(whereClause);
+  updateFeatureCount();
```

Comment:
❓ 
Is this needed? Changing the definition expression on the feature layer has no effect on the number of features in a given extent.

Reply:
I believe this should stay, this line emphasizes that the feature count in extent should not change with definition expressions unless the extent and visible features change. 

If the user changes the extent and reapplies definition expression, the feature count number should still update

Reply:
That's a good point. In that case, I would consider adding a comment so that is clear. Thanks for the explanation.

Optional.